### ISSUE TITLE
Windows App Temporarily Shows Old Window Style

### ISSUE THREAD

**Issue by TheSoloJake** [toggl-open-source/toggldesktop#4576]
### 💻 Environment

Platform: Windows 
OS Version: Windows 10
Toggl Version: 7.5.295

### 🐞 Actual behavior
When opening the app or minimizing the window, the app temporarily appears to revert to an older windows app frame before immediately correcting itself. It does not appear to hamper the app, but is simply more of a weird UI look. 

### 📦 Additional info
User [report](https://app.intercom.com/a/apps/ayixs927/inbox/inbox/2137598/conversations/106481304718682) (contains screenshots detailing issue)

**Comment by skel35** [comment]
I've seen this issue on my PC, which got fixed by the Windows update. We're unlikely to prioritize it due to it being dependent on a particular Windows update.

**Comment by TheSoloJake** [comment]
@skel35 can you share which "Windows update" you're referring to? If it's something I can recommend to the user I'll do that

**Comment by skel35** [comment]
Just one of the regular updates to my Windows 10 Pro 20H1. I've had a few updates since then, so I'm afraid I won't be able to point to a specific update.
I've just updated my OS to the Windows 10 Pro 20H2 (build 19042.572) and I'm not experiencing such an issue.

**Comment by VeljkoTN** [comment]
@skel35 this user is on version 20H2, OS Build 19042.610, Windows insider beta channel - could that be the issue? the fact that they are on a beta?

**Comment by skel35** [comment]
No, I don't think so. Then it's probably not related to the specific Windows update.
It might be the quirks of the `MahApps.Metro` library we're using, we'll look into it at some point if we continue using this library.
